python/cudf/cudf/core/column/numerical.py

```diff
@@ -107,7 +111,7 @@ def normalize_binop_value(self, other):
             if other_dtype == np.dtype("float16"):
                 other = np.dtype("float32").type(other)
                 other_dtype = other.dtype
-            if other_dtype.kind in "u":
+            if self.dtype.kind in "b":
```

Comment:
To be in sync with pandas, as pandas returns signed output. If we don't changes the type, it will return unsigned types.

https://github.com/rapidsai/cudf/blob/d73f7ae492dfaf691b08dcca566a48c8a21ad4f0/python/cudf/cudf/tests/test_binops.py#L659